### ISSUE TITLE
ref/mod: vcs-find section should mention the "mod" version control system

### ISSUE THREAD

**Issue by mvdan** [golang/go#48867]
I recently asked a question which boils down to: can I publish a Go module in plain zip archives, without using a standard VCS like git?

@bcmills helpfully pointed me towards https://golang.org/ref/mod#serving-from-proxy, which is basically what I want :)

The reason I had failed to find that section of the reference doc is because I looked in https://golang.org/ref/mod#vcs-find, which does not list "mod" in its table. It probably should, and have the mention link to the serving-from-proxy section.

**Comment by dmitshur** [comment]
In that section, where the 3 parts of go-import are being explained, the following written for "vcs":

> `vcs` is the version control system. It must be one of the tools listed in the table below or the keyword mod, which instructs the go command to download the module from the given URL using the [GOPROXY protocol](https://go.dev/ref/mod#goproxy-protocol). See [Serving modules directly from a proxy](https://go.dev/ref/mod#serving-from-proxy) for details.

That seems pretty close to what you described, the key difference being that "mod" is covered as a special case before the table, and so it's not a part of the table as a result. Trying to refactor it to be a part of the table would require coming up with something to say for the 'GOVCS default' and 'Secure schemes' columns, which I'm not sure apply in the same way as for actual VCS commands.

Do you think it's worth pursuing making it a row in the table anyhow?

**Comment by mvdan** [comment]
Yes. "Serving modules directly from a proxy" is not a title that I went for directly, because I did not want to use a Go modules proxy at all, I just wanted to serve zip files over HTTP. I went for https://go.dev/ref/mod#vcs-find because I was looking for some way for the upstream to provide a "version control system" which is just flat archive files over HTTP, and indeed, `mod` is treated as one more VCS system:

> If the version control system is mod, [...]

It may be enough to add a reminder below the VCS table, like:

> `mod` is also a valid `vcs`, which allows serving module sources as archive files without one of the VCS systems listed above. See (previous section).

Although I would argue that "Serving modules directly from a proxy" needs to be refactored. It is under the wrong section; it describes the `mod` VCS, which is related to proxies, but that needs to be underneath the "Version Control Systems" section, really. Proof of that is that it needs to explain how `<meta>` tags work, which is explained in detail once again once the VCS section begins.
